src/style.css

```diff
@@ -0,0 +1,112 @@
+h1 {
```

Comment:
Nice job with the styling, great work, adding some box shadow effect to your main container can make your to-do list look more aesthetically pleasing

Reply:
Yes I think, as of now, the application seems very dull. I need to improve the styling component of the project. Ill definitely work on styling this weekend. When I am done, Ill reach out to you for another feedback. 
Stay Safe ✌️